backend/presentation/paginations.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class PresentationAndTagSetPagination(PageNumberPagination):
```

Comment:
I would suggest to rename the class. What if you would like to add `pagintion_class` to another `ViewSet?` Will you rename it to `PresentationAndTagAndAnotherModelSetPagination`?
```suggestion
class StandardResultsSetPagination(PageNumberPagination):
```